### PR TITLE
missing lemmas about ereal

### PR DESCRIPTION
**Pull request by affeldt-aist**
##### Motivation for this change

just noticed a missing lemma in `constructive_ereal.v` that helps shorten a proof

##### Checklist

- [x] added corresponding entries in `CHANGELOG_UNRELEASED.md`

~~- [ ] added corresponding documentation in the headers~~

Reference: [How to document](https://github.com/math-comp/math-comp/wiki/How-to-document)

##### Reminder to reviewers

- Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-creating-and-review-PRs)
- Put a milestone if possible
- Check labels

### CODE REVIEW

**Review by proux01 — COMMENTED** [review]
We need the dual addition versions but otherwise looks good to me.

**Comment by affeldt-aist** [comment]
The dual lemmas have not been updated to the new naming scheme (my fault).
Shall we rename, say, `lee_dadd` to `leedD` or `lee_dD` (maybe more readable for lemmas with longer suffixes)?

**Comment by proux01** [comment]
I agree the latter is more readable.

**Comment by affeldt-aist** [comment]
Then I'll do the renaming along this PR.

**Comment by proux01** [comment]
Thanks a lot!

**Comment by affeldt-aist** [comment]
Lemmas in the dual version should be aligned.

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -39,6 +39,10 @@
 - in `realfun.v`:
   + lemma `nondecreasing_at_left_is_cvgr`
 
+- in `constructive_ereal.v`:
+  + lemmas `lteD2rE`, `leeD2rE`
+  + lemmas `lte_dD2rE`, `lee_dD2lE`
```
Comment (proux01) [inline CHANGELOG_UNRELEASED.md:44]:
```suggestion
  + lemmas `lte_dD2rE`, `lee_dD2rE`
```

File: CHANGELOG_UNRELEASED.md [hunk 2]
```diff
@@ -85,15 +89,52 @@
...
+  + `lee_daddr` -> `lee_dDr`
+  + `gee_daddl` -> `gee_dDl`
+  + `gee_daddr` -> `gee_dDr`
+  + `lte_daddl` -> `lte_dDl`
+  + `lte_daddr` -> `lte_dDr`
+  + `gte_dsubl` -> `gte_dBl`
+  + `gte_dsubr` -> `gte_dBr`
+  + `gte_daddl` -> `gte_dDl`
+  + `gte_daddr` -> `gte_dDr`
+  + `lte_dadd2lE` -> `lte_dD2lE`
```
Comment (proux01) [inline CHANGELOG_UNRELEASED.md:104]:
```suggestion
  + `lee_dadd2lE` -> `lee_dD2lE`
  + `lte_dadd2lE` -> `lte_dD2lE`
```

**Review by proux01 — APPROVED** [review]
A few details but otherwise LGTM

**Pull request merged**